source/faq/questions/using-cypress-faq.md

```diff
@@ -600,3 +600,30 @@ cy.get('@consoleLog').should('be.calledWith', 'Hello World!')
 ```
 
 Also, check out our {% url 'Stubbing `console` Receipe' recipes#Stubbing-and-spying %}.
+
+## {% fa fa-angle-right %} How do I use special characters with cy.get?
+
+[According to the CSS spec](https://www.w3.org/TR/html50/dom.html#the-id-attribute), special characters like `/`, `.` are valid characters for ids. 
```

Comment:
We use special url tags that check each url to ensure there aren't any dead links.

```suggestion
{% url "According to the CSS spec" https://www.w3.org/TR/html50/dom.html#the-id-attribute %}, special characters like `/`, `.` are valid characters for ids. 
```